src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
```

Comment:
Avoid variable namings like `resultObject` as it decreases code readability. Instead, use a name that reflects the content or purpose of the object, such as `styleObject` or `cssProperties`. [CODE STYLE]